src/components/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import "../Footer.scss"
+
+export function Footer() {
+
+    return (
+        <div className = "footer-main">
+                <section className="footer-left"></section>
+                <section className="footer-right"></section>
```

Comment:
Please either fill these out or put placeholder text in them - prefer not to have empty tags